src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -5936,7 +5936,6 @@ class LibraryServiceClient {
    * });
    */
   streamShelves(request, options) {
-    request = request || {};
```

Comment:
For server streaming we can still have it :)

Reply:
Got it, thanks.

Reply:
Before pulling new updates from master, all the tests passed, while now seems bazel_test starts to fail.

Reply:
It's flaky, @vam-google maybe will fix it when he gets back from his vacation.

Reply:
Sounds cool, then we can hold this until the tests are fixed. Thanks

Reply:
No. We just merge it if the bazel test is the only one that fails.